lib/experimental/fonts/font-library/class-wp-font-family.php

```diff
@@ -300,12 +300,17 @@ private function sanitize() {
 			'version'  => '2',
 			'settings' => array(
 				'typography' => array(
-					'fontFamilies' => array( $this->data ),
+					'fontFamilies' => array(
+						'custom' => array(
```

Comment:
The `custom` key is necessary to accommodate the input expected by `remove_insecure_properties`.